docs/src/jsx/allDemos.stories.js

```diff
@@ -50,6 +52,7 @@ const allDemos = {
   SpheresSingle,
   Text,
   Triangles,
+  DuckScene: () => <DuckScene modelURL={duckModel} />,  // eslint-disable-line
```

Comment:
nit: might be nice to use `eslint-disable-line <eslint-rule-that-fails>` as a small form of documentation as to what lint rule you're opting out of.